crates/precompile-native-currency/src/lib.rs

```diff
@@ -211,7 +213,7 @@ where
             owner.into(),
             spender.into(),
             amount.try_into().map_err(|_| PrecompileFailure::Error {
-                exit_status: ExitError::Other("value is out of bounds".into()),
+                exit_status: ExitError::Other("allowance convertation error from U256".into()),
```

Comment:
```suggestion
                exit_status: ExitError::Other("allowance is out of bounds".into()),
```

Reply:
Actually, how about actually matching the error here, or maybe even doing a `.to_string`?

Reply:
I would suggest to use explicit `allowance is out of bounds` here as default error message for `TryFrom<U256>` is just `Overflow` without any details.

Reply:
https://github.com/humanode-network/humanode/pull/809/commits/e9d0e8cfadfb50cd099611d55745e195fcd2d266